packages/camera/camera_android_camerax/example/lib/camera_preview.dart

```diff
@@ -63,10 +63,10 @@ class CameraPreview extends StatelessWidget {
 
   int _getQuarterTurns() {
     final Map<DeviceOrientation, int> turns = <DeviceOrientation, int>{
-      DeviceOrientation.portraitUp: 0,
-      DeviceOrientation.landscapeRight: 1,
-      DeviceOrientation.portraitDown: 2,
```

Comment:
I'm not sure what to do about portraitDown. How is it simulated?

Reply:
Do you have a Samsung devices? I can't see it on my pixel but I have seen it on a Galaxy IIRC

Reply:
I tested two Galaxys and they did not rotate to portraitDown :') It could have possibly been a Samsung tablet and I can definitely help test out different devices in the office, but I'm not sure, sorry.

Reply:
maybe it will be the same as portraitUp?

Reply:
Oh I assumed it would be 1 because it should face opposite of portraitUp?

Reply:
Meaning that the change has just shifted what is considered the natural orientation

Reply:
makes sense!